full-service/src/service/sync.rs

```diff
@@ -52,11 +53,18 @@ pub struct SyncThread {
 }
 
 impl SyncThread {
-    pub fn start(ledger_db: LedgerDB, wallet_db: WalletDb, logger: Logger) -> Self {
+    pub fn start(
+        ledger_db: LedgerDB,
+        wallet_db: WalletDb,
+        accounts_with_deposits: Arc<Mutex<HashMap<AccountID, bool>>>,
+        _restart: Arc<Mutex<AtomicBool>>, // FIXME: when does this get set and by what?
```

Comment:
⚠️ I'm not sure about the actual implementation of this piece of it cc @holtzman